homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
```

Comment:
I would strongly recommend to use entity descriptions in the binary sensor platform

Reply:
switch could also use them